File: bootstrap/v0.0.7.5/benchmarks/rust/fibonacci.rs
fn fibonacci(n: i32) -> i32 {
    if n < 2 {
        n
    } else {
        let a = fibonacci(n - 1);
        let b = fibonacci(n - 2);
        a + b
    }
}

fn main() {
    let result = fibonacci(35);
    std::process::exit(result);
}

File: bootstrap/v0.0.8.2/benchmarks/java/Fibonacci.java
public class Fibonacci {
    public static int fibonacci(int n) {
        if (n < 2) {
            return n;
        } else {
            int a = fibonacci(n - 1);
            int b = fibonacci(n - 2);
            return a + b;
        }
    }

    public static void main(String[] args) {
        int result = fibonacci(35);
        System.exit(result);
    }
}